docs/device-agent/quickstart.md

```diff
@@ -8,60 +8,100 @@ meta:
 
-### Prerequisites
+![FlowFuse Device Agent Workflow](./images/device-agent-workflow.png){data-zoomable}
+_Workflow to onboard new Edge Devices through the FlowFuse Platform and Device Agent_
 
-If you do not already have the device agent installed, follow the [install instructions](./install.md), otherwise, proceed to the next step.
+## Setup & Installation
 
-_NOTE: It is good practice to keep your Device Agent up to date. Update instructions are also provided in the link above._
+### Step 1: Install Device Agent (On Device)
 
-_NOTE: If you prefer to use the Device Agent WEB UI, follow the [Quick Start with Web UI](./quickstart-web-ui.md) guide._
+NodeJS of version 20 or later is recommended, through the device agent supports NodeJS v14 or later.
+
+#### Linux/MacOS
+
+```bash
+sudo npm install -g @flowfuse/device-agent
+```
+
+#### Windows
+
+```bash
+npm install -g @flowfuse/device-agent
+```
+
+For full install instructions, follow the [install instructions here](/docs/device-agent/install.md).
+
+### Step 2: Register Device (On FlowFuse)
 
-### Step 1: Add a new Device
 1. Open your web browser and go to your FlowFuse application page.
 2. Navigate to the **Devices** section.
 3. Click **Add Device.**
 4. Fill in the **Device Name** & **Device Type** and select an **Application**
 
-    <img src="images/add_device.png" />
+    ![Screenshot of the dialog form to fill out when registering a Device](./images/add_device.png){data-zoomable}
+    _Screenshot of the dialog form to fill out when registering a Device_
+    
 5. Click **Add**
 6. You will be presented with a **Device Configuration** that you will need in the next step. _(Do not close this window just yet.)_
 
-    <img src="images/config_yml2a.png"/>
+    ![Screenshot of the dialog shown to a suer when a Device is registered](./images/config_yml2a.png){data-zoomable}
+    _Screenshot of the dialog shown to a suer when a Device is registered_
+
+### Step 3: Connect the Device (On Device)
+
+1. Open a Terminal/Command Prompt on the Device.
+2. Paste the command presented when you registered your Device, and run it.
 
-### Step 2: Connect the Device
-1. Open a Terminal/Command Prompt.
-2. Paste the command copied from the previous step and run it.
+    ![Example output in a Terminal showing a device agent successfully connecting to the FlowFuse Platform](./images/device_cli.png){data-zoomable}
+    _Example output in a Terminal showing a device agent successfully connecting to the FlowFuse Platform_
+
+### Step 4: Start the Device Agent (On Device)
+
+1. Start the Device Agent using the command instructed by the device agent output e.g:
 
-    <img src="images/device_cli.png" />
-3. Start the Device Agent using the command instructed by the device agent output e.g:
    ```bash
    flowfuse-device-agent
    ```
 
 Your device is now connected to FlowFuse and ready to be used.
 
+## Deploy Flows to Device
+
+There are two approaches to running flows on your Devices.
```

Comment:
```suggestion
There are two approaches to deploying flows to your Devices.
```

This section header is "Deploy Flows to Device" - the original text stated "running flows on your Devices" which when i read it, I was thinking we were referring to how the device-agent runs the node-red flows (like as a service or in docker)